openlibrary/catalog/update_count.py

```diff
@@ -1,7 +1,7 @@
-from olwrite import Infogami, add_to_database
+from openlibrary.catalog.olwrite import Infogami, add_to_database  # noqa: F401
 import web
 import dbhash
-from read_rc import read_rc
+from openlibrary.catalog.read_rc import read_rc
```

Comment:
Anything that imports this `read_rc` is probably a standalone script and can be removed, along with `read_rc` itself- the `~/.olrc` config does not exist and most things that use it were removed in the last part of the catalog clean up.

Reply:
I reverted the `read_rc()` files to ease the review process.

Reply:
https://github.com/internetarchive/openlibrary/pull/3053#issuecomment-595170889